### ISSUE TITLE
Tile is not a legal element

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#8547]
The Carbon [tile's coding guidance](https://www.carbondesignsystem.com/components/tile/code) shows a construct that is not a legal element in html:
`<Tile>`
      Default tile
`</Tile>`

There is no tile element. There is also no established role for a tile, in html or aria.

As a custom component, a tile must be created through divs, which are in turn given correct semantic roles. 
A possible approach could be to use https://w3c.github.io/aria/#aria-roledescription

`<div role="group" aria-roledescription="tile">`

**Comment by mbgower** [comment]
It helps my context, but it doesn't help a team construct something that is accessible.

I ended up on the Carbon site looking at this because I'm seeing a bunch of end user confusion: blind users complaining about the context or lack thereof, designers unclear how to label the tiles (are they headings, are they anchor names, is it a name for the tile itself), trying to decide how the roles change when the tile is read-only, versus an actionable component, versus a static component containing links as part of its content, etc.

I was going to open up that as a separate issue trying to tackle the guidance and accessibility consideration, but then I looked at the code, saw the tile element, and started with that :)

is the best course here to close this, and move the discussion into a separate issue?

I'm not sure if the lack of listed 'properties' of the react tile is a contributor to the problems I'm encountering or what.

**Comment by tw15egan** [comment]
The `Tile` component itself is made up of 4 types: 

- a plain `Tile` ([storybook](https://react.carbondesignsystem.com/iframe.html?id=components-tile--default)), like the one listed above, is basically just a simple div container that may contain an actionable item or a heading inside of it, but the tile itself has no actionable items.
- a `ClickableTile` ([storybook](https://react.carbondesignsystem.com/iframe.html?id=components-tile--clickable)), which renders an anchor element under the hood and (should) behave like a link out of the box
- an `ExpandableTile` ([storybook](https://react.carbondesignsystem.com/iframe.html?id=components-tile--expandable)), which allows a user to render content on click. I see that this has no roles or anything associated with it, so perhaps this is a variant this is causing confusion? I do not see any DAP errors, but definitely open to any suggestions on how we can improve this
- `SelectableTile` ([storybook](https://react.carbondesignsystem.com/iframe.html?id=components-tile--multi-select)), which  lets a user select multiple tiles. I see here in the storybook that we are wrapping them in a `<div role="group" aria-label="selectable tiles">`, but it is not included in the component OOTB, so this could definitely be improved upon. Removing that `div` results in a `Checkbox input and others with the name "tiles" are not grouped together` IBM 1.3.1 Info and relationships violation
- `RadioTile` is similar to `SelectableTile`, but only allows one selection. This _does_ come wrapped in a `TileGroup` component to satisfy a11y requirements for radio buttons


I think it may be best to 1) identify which variants have a11y concerns and 2) creating issues specific to those variants and which a11y violations/improvements need to be addressed.

What do you think?

**Comment by joshblack** [comment]
Going to close this out due to inactivity 👍
